src/queries/eventRecords.js

```diff
@@ -31,12 +40,58 @@ export const GET_EVENT_RECORD = gql`
           url
         }
       }
+      addresses {
+        city
+        street
+        zip
+      }
+      contact: contacts {
```

Comment:
We have to be careful here. One `contact` in `PointOfInterest` cannot be handled the same way like multiple `contacts` for `EventRecord`. One is an Object and the other an Array of Objects.